src/igs_network.c

```diff
@@ -4109,7 +4115,14 @@ void igsagent_set_name (igsagent_t *agent, const char *name)
                                                 + strlen ("-IGS-SERVICES") + 1);
     snprintf (agent->services_channel, IGS_MAX_AGENT_NAME_LENGTH + 15,
               "%s-IGS-SERVICES", agent->definition->name);
-
+    
+    if (agent->igs_channel)
+        free (agent->igs_channel);
+    agent->igs_channel = (char *) zmalloc (strlen (agent->definition->name)
+                                                + strlen ("-IGS") + 1);
+    snprintf (agent->igs_channel, IGS_MAX_AGENT_NAME_LENGTH + 5,
```

Comment:
Use strlen() instead of magic numbers to avoid mistakes.  
`IGS_MAX_AGENT_NAME_LENGTH + strlen("-IGS") + 1`  
I also find it more readable this way.